docs/API.Auth.GeneratePassword.md

```diff
@@ -0,0 +1,30 @@
+# Function: *Generate Password* #
+
+## Description ##
+
+When creating a new CM-Well user, you can use the **generate-password** API to generate a random password and its basic-encoded value. This password can be changed later if required (see [Change Password API](API.Auth.ChangePassword.md)).
+
+>**Note:** You will need an access token with admin permissions to call this API.
```

Comment:
Actually, since it's not affecting anything in CM-Well, there's no need for an Admin token here. Anyone can generate-password as much as they like - only when used later in a UserInfoton it has some meaning. This API is just a "helper method" to invoke the bcrypt algorithm on a random value with CM-Well Cluster's private key.